src/ng/directive/ngShowHide.js

```diff
@@ -182,6 +182,23 @@ var NG_HIDE_IN_PROGRESS_CLASS = 'ng-hide-animate';
       });
     </file>
   </example>
+ *
+ * @knownIssue
+ *
+ * ### Flickering when using ngShow to toggle between elements
+ *
+ * When using {@link ngShow} and / or {@link ngHide} to toggle between elements , it can
+ * happen that both the element to show and the element to hide are visible at the same time.
```

Comment:
I would mention that this only happens for a brief moment; it is not a permanent state.
It might also be worth mentioning that it affects some browsers (_\*cough\*_ IE _\*cough\*_) more than others.